Sources/TuistKit/Services/RunService.swift

```diff
@@ -0,0 +1,142 @@
+import Foundation
+import RxSwift
+import TSCBasic
+import TuistAutomation
+import TuistCore
+import TuistSupport
+
+enum RunServiceError: FatalError {
+    case appNotFound(targetName: String)
+    case simulatorNotFound
+    case targetNotFound(forSchemeName: String)
+
+    var description: String {
+        switch self {
+        case let .appNotFound(targetName):
+            return "Couldn't find any apps built for \(targetName)"
+        case .simulatorNotFound:
+            return "Could't find simulator"
+        case let .targetNotFound(forSchemeName):
+            return "Couldn't find \(forSchemeName)"
+        }
+    }
+
+    var type: ErrorType {
+        switch self {
+        case .appNotFound, .simulatorNotFound, .targetNotFound:
+            return .abort
+        }
+    }
+}
+
+final class RunService {
+    let simulatorController: SimulatorControlling
```

Comment:
Nitpick: I'd remove the empty lines between the definition of the instance variables.